library/std/src/io/error.rs

```diff
@@ -373,8 +373,8 @@ pub enum ErrorKind {
     TooManyLinks,
     /// A filename was invalid.
     ///
-    /// This error can also cause if it exceeded the filename length limit.
-    #[unstable(feature = "io_error_more", issue = "86442")]
+    /// This error can also occur if the filename length limit was exceeded.
```

Comment:
E.g. `gethostname` can also [return `ENAMETOOLONG`](https://www.man7.org/linux/man-pages/man2/gethostname.2.html#ERRORS), so I think we should phrase the documentation in a way that is a bit less file-specific.
```suggestion
    /// This error can also occur if a length limit for a name was exceeded.
```